packages/wouter/src/use-hash-location.js

```diff
@@ -22,11 +22,31 @@ const subscribeToHashUpdates = (callback) => {
 // leading '#' is ignored, leading '/' is optional
 const currentHashLocation = () => "/" + location.hash.replace(/^#?\/?/, "");
 
-export const navigate = (to, { state = null } = {}) => {
+export const navigate = (to, { state = null, replace = false } = {}) => {
   // calling `replaceState` allows us to set the history
   // state without creating an extra entry
   const [hash, search] = to.replace(/^#?\/?/, "").split("?");
 
+  if (replace) {
+    history.replaceState(
```

Comment:
Not sure but, maybe we can always fire this event manually and that way we won't need to do the assigment and can simplify the code?